src/main/java/org/folio/marccat/dao/CatalogDAO.java

```diff
@@ -97,19 +104,21 @@ private void loadHeading(final AccessPoint tag, final int userView, final Sessio
   public void saveCatalogItem(final CatalogItem item, final Session session) throws HibernateException {
 
     final Transaction transaction = getTransaction(session);
-    final String myView = makeSingleViewString(item.getUserView());
     final ItemEntity itemEntity = item.getItemEntity();
     if (!itemEntity.isNew()) {
       itemEntity.setUpdateStatus(UpdateStatus.CHANGED);
     }
     persistByStatus(itemEntity, session);
     final List<Tag> tagList = item.getTags().stream().map(aTag -> {
+      String myView = makeSingleViewString(item.getUserView());
```

Comment:
The degree of complexity with these two instructions is increased from 30 to 32 in this method. It is preferable to fix it.